README.md

```diff
@@ -1,84 +1,93 @@
+Click on the **Services** subtab and click **Create service**.
+Select the `vision` type, then select the `detector:facial-detector` model. 
+Click **Add module**, then enter a name for your facial detector and click **Create**.
 
-The following attributes may be configured as facial-detector config attributes.
-For example: the following configuration would use the `ssd` framework:
+On the new component panel, copy and paste the following attribute template into your vision service's **Attributes** box:
 
-``` json
+```json 
 {
-  "detection_framework": "ssd"
+  "detection_framework": "<detection-framework>",
+  "recognition_model": "<recognition-model>",
+  "face_labels": {
+    "matt": "<path-to-image-1>",
+    "suzy": "<path-to-image-2>"
+  },
+  "verify_threshold": <verify-threshold>,
+  "disable_detect": <disable-detect>,
+  "disable_verify": <disable-verify>
 }
 ```
 
-### detection_framework
-
-*enum - "opencv"|"retinaface"|"mtcnn"|"ssd"|"dlib"|"mediapipe"|"yolov8" (default: "ssd")*
-
-The detection framework to use for facial detection.  `ssd` is chosen as the default for a good balance of speed and accuracy.
-
-### recognition_model
+> [!NOTE]
+> For more information, see [Configure a Machine](https://docs.viam.com/manage/configuration/).
 
-*enum -   "VGG-Face"|"Facenet"|"Facenet512"|"OpenFace"|"DeepFace"|"DeepID"|"ArcFace"|"Dlib"|"SFace" (default: "ArcFace")*
+### Attributes 
 
-The model to use for facial recognition.  `ArcFace` is chosen as the default for a good balance of speed and accuracy.
+The following attributes are available for `vision:facial-detector:ssd` facial-detectors:
 
-### face_labels
+| Name   | Type | Inclusion | Description| Supported |
+|--------|-------|----------|------------|-----------|
+| `detection_framework` | enum | Optional| The detection framework to use for facial detection.<br> Default: `ssd`, for a good balance of speed and accuracy. | `"opencv"`, `"retinaface"`, `"mtcnn"`, `"ssd"`, `"dlib"`, `"mediapipe"`, `"yolov8"` |
+| `recognition_model` | enum | Optional| The model to use for facial recognition.<br> Default: `ArcFace`, for a good balance of speed and accuracy.| `"VGG-Face"`, `"Facenet"`, `"Facenet512"`, `"OpenFace"`, `"DeepFace"`, `"DeepID"`, `"ArcFace"`, `"Dlib"`, `"SFace"` |                     |
+| `face_labels` | object  | Optional | An object map of `key:label, value:path` for matching against reference face images. | - |
+| `verify_threshold` | number  | Optional | If `disable_verify` is `false` and `face_labels` are set, this threshold must be met or exceeded for a verification match.<br> Default: `0.8` | - |
+| `disable_detect` | bool | Optional| If `false`, any faces detected but not verified as matching a label will be labeled as "face".<br> Default: `false` | - |
+| `disable_verify` | bool | Optional | If `false` and `face_labels` are set, will attempt to verify any faces detected. If you only want verified faces, set this and `disable_detect` to `true`.<br> Default: `false` | - |
```

Comment:
Could otherwise sound like false is set
```suggestion
| `disable_detect` | bool | Optional| If `false`, any faces detected but not verified as matching a label will be labeled as "face".<br> Default: `false` | - |
| `disable_verify` | bool | Optional | If set to `false` and `face_labels` are set, will attempt to verify any faces detected. If you only want verified faces, set this and `disable_detect` to `true`.<br> Default: `false` | - |
```